iocage/lib/Config/Jail/Properties/Addresses.py

```diff
@@ -154,6 +169,37 @@ def add(
         if notify:
             self.__notify()
 
+    def _add(
+        self,
+        nic: str,
+        addresses: typing.Optional[
+            typing.Union[
+                typing.List[str],
+                str,
+                typing.List[ipaddress.IPv4Interface],
+                ipaddress.IPv4Interface,
+                typing.List[ipaddress.IPv6Interface],
+                ipaddress.IPv6Interface
+            ]
+        ]=None,
+        notify: bool=True
+    ) -> None:
+
+        if isinstance(addresses, list) is False:
+            self.add(nic=nic, addresses=[addresses], notify=notify)
+            return
+
+        _addresses: typing.List[IPAddressInput] = []
+        try:
+            _addresses = [self.ADDRESS_CLASS(address) for address in addresses]
+        except ipaddress.AddressValueError as e:
```

Comment:
i think this should simply be ValueError, at least according to the documentation

Reply:
```
>>> import ipaddress
>>> ipaddress.IPv6Interface("192.168.23.1/24")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/ipaddress.py", line 2077, in __init__
    IPv6Address.__init__(self, addr[0])
  File "/usr/local/lib/python3.6/ipaddress.py", line 1917, in __init__
    self._ip = self._ip_int_from_string(addr_str)
  File "/usr/local/lib/python3.6/ipaddress.py", line 1654, in _ip_int_from_string
    raise AddressValueError(msg)
ipaddress.AddressValueError: At least 3 parts expected in '192.168.23.1'
```